_data/social.yml

```diff
@@ -34,7 +34,7 @@ websites:
       tfa: Yes
       sms: Yes
       doc: https://www.biggerpockets.com/forums/25/topics/232201
-     
```

Comment:
Prettier must have detected these spaces and deleted them.

Reply:
Hey @eetrain, would you be so kind as to revert that single line change, so that your PR only includes the Eventbrite change and the accompanying newline?
We will clear out superfluous characters at a later date.
Thanks!
// Kai

Reply:
@kmpoppe 
No problem. As requested, this PR now only includes the Eventbrite change.